### PR TITLE
Add privacy url and handler

### PR DESCRIPTION
**Pull request by dracco1993**
## Description
Add `/privacy` url and handler

## Motivation and Context
More `py3`

## How Has This Been Tested?
Locally

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

File: ops/check_venv_deps.py [hunk 1]
```diff
@@ -12,9 +12,7 @@
 def test_requirements():
     missing_requirements = []
     for requirement_file in REQUIREMENTS_FILES:
-        requirements = pkg_resources.parse_requirements(
-            requirement_file.read_text()
-        )
+        requirements = pkg_resources.parse_requirements(requirement_file.read_text())
```
Comment (dracco1993) [inline ops/check_venv_deps.py:15]: Uh, the auto-linter got this one.  👀😂

**Comment by dracco1993** [comment]
In addition to adding the stuff for the `/privacy` page, I bumped up the cache time to `1 week` for this and the past couple of pages, since they're static/redirects; if this isn't a good idea, or I should use something else, let me know.   👍

File: ops/check_venv_deps.py (hunk 1 above)
Comment (phil-lopreiato) [reply ops/check_venv_deps.py:15]: Whoops!

**Comment by phil-lopreiato** [comment]
Yeah, a 1w cache time makes sense for this + the others

**Pull request merged**